topics/using-github-actions.dita

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE task PUBLIC "-//OASIS//DTD DITA General Task//EN" "generalTask.dtd">
+<!--  This file is part of the DITA Open Toolkit project. See the accompanying LICENSE file for applicable license.  -->
+<task id="ID">
+  <title>Running the <cmdname>dita</cmdname> command from a GitHub Action</title>
+  <titlealts>
+    <navtitle>Using GitHub Actions</navtitle>
+  </titlealts>
+  <shortdesc>
+    <ph id="github-actions-desc">
+      <xref
+        keyref="github-actions"
+      /> are a CI/CD Workflow mechanism attached to GitHub. Each Action is an individual unit of functionality that can be combined with other GitHub Actions to create workflows, which are triggered in response to certain GitHub events. As of version 3.6, the DITA-OT project provides an official <codeph
+      >dita-ot-build</codeph> Action that can be used as a step within a GitHub workflow to create documentation as part of your CI/CD pipeline</ph></shortdesc>
+  <prolog>
+    <metadata>
+      <keywords>
+        <indexterm><cmdname>dita</cmdname> command
+          <indexterm>running from GitHub CI/CD</indexterm></indexterm>
+        <indexterm>GitHub Actions</indexterm>
+      </keywords>
+    </metadata>
+  </prolog>
+  <taskbody>
+    <section>
+      <title>About GitHub Actions</title>
+      <p>
+        GitHub Actions can automate tasks such as document generation as part of your
+        software development life cycle. GitHub Actions are event-driven, allowing a
+        series of tasks to run one after another when a specified event has occurred.
+      </p>
+      <p>
+        Each step is an individual atomic task that can run commands in a job. A step can be either an action or a shell command. Each step in a job executes on the same runner, allowing the actions in that job to share data with each other, therefore files generated through the <codeph
+        >dita-ot-build</codeph> Action can be archived or published by later actions within the same job.
+    </p>
+    </section>
+    <steps>
+      <step>
+        <cmd>In your GitHub repository, create the <codeph
+          >.github/workflows/</codeph> directory to store your workflow files.</cmd>
+      </step>
+      <step>
+        <cmd>In the <codeph
+          >.github/workflows/</codeph> directory, create a new file called dita-ot-build-actions.yml and add the following code.</cmd>
+          <stepxmp>
+          <codeblock outputclass="syntax-yaml">name: CI
+'on':
+  push:
+    branches:
+      - master
+jobs:
+  build-dita:
+    name: Build DITA
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+          </codeblock>
```

Comment:
Would it make sense to add this file to the `samples/` folder along with the others, and pull in the relevant lines here with [`line-range` pointers](https://www.dita-ot.org/dev/reference/extended-functionality.html#code-reference__coderef-charset)?

This would make it easier to validate, auto-format, and update the YAML if necessary.

Reply:
Fixed 86c44d3